### DISCUSSION TITLE
iptable error while installing Lamp on ubuntu (installed in windows 10)

### DISCUSSION THREAD

**Discussion by jiyang666** [microsoft/WSL#8255] [General]
error:
ERROR: problem running iptables: iptables v1.8.4 (legacy): can't initialize iptables table `filter': Table does not exist (do you need to insmod?)
Perhaps iptables or your kernel needs to be upgraded.

Thanks in advance

**Comment by jiyang666** [comment]
![图片](https://github.com/JiYang1028/video-tools/blob/master/QQ%E5%9B%BE%E7%89%8720220408184330.png?raw=true)

**Comment by Biswa96** [comment]
Please fill out the issue template. From the screenshot, it looks like you are using WSL1. Try with converting the distribution to WSL2. This docs https://docs.microsoft.com/en-us/windows/wsl/install may help.
